packages/unmock-node/src/types.ts

```diff
@@ -4,4 +4,5 @@ export {
   ISerializedRequest as UnmockRequest,
   ISerializedResponse as UnmockResponse,
   ServiceStoreType as UnmockServices,
+  IService as Service,
```

Comment:
Maybe `UnmockService` or similar?

Reply:
Does `Service` collide with something or why `UnmockService`? Or would you like to keep it consistent with `UnmockRequest` and `UnmockResponse`? Not sure where to draw the line then, should everything be prefixed with `UnmockSomething`? It also feels a bit extra to write `const github: UnmockService = ...` instead of `const github: Service = ...` but I can of course change it.

Reply:
Exported both so one can choose whichever!

Reply:
It does feel a bit extra to write that. It doesn't collide with anything AFAICS, but more so that `Service` is quite general, and does not necessarily imply that it's related to unmock library.

We could think of a different prefix of course. Perhaps, for example, a lowercase `u`? :grin: